From 5aae4f699578fd1c23970a0afd55fde94b2c21cb Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Thu, 1 Jun 2017 13:03:58 +0500
Subject: [PATCH 2/5] vktrace: do not link to internal loader

The binaries are being linked to the internally
generated libvulkan.so which causes rpath issues
at the end.
We fix this by linking to the originally generated
lib through vulkan-loader-layers package.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 vktrace/vktrace_extensions/vktracevulkan/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vktrace/vktrace_extensions/vktracevulkan/CMakeLists.txt b/vktrace/vktrace_extensions/vktracevulkan/CMakeLists.txt
index 60e4cc2d7..d961edfc7 100644
--- a/vktrace/vktrace_extensions/vktracevulkan/CMakeLists.txt
+++ b/vktrace/vktrace_extensions/vktracevulkan/CMakeLists.txt
@@ -24,7 +24,7 @@ endif()
 if (${CMAKE_SYSTEM_NAME} MATCHES "Linux" OR
     ${CMAKE_SYSTEM_NAME} MATCHES "Darwin" )
     set(VKTRACE_VULKAN_LIB
-        ${CMAKE_BINARY_DIR}/loader/lib${API_LOWERCASE}.so
+        libvulkan.so
     )
 
 endif()
-- 
2.11.1

